docs-ref-services/batch.md

```diff
@@ -59,4 +53,53 @@ new_pool.virtual_machine_configuration = vmc
 client.pool.add(new_pool)
 ```
 
-Explore more [sample Python code](https://azure.microsoft.com/resources/samples/?platform=python) you can use in your apps.
\ No newline at end of file
+## Management API
+Use the Azure Batch management libraries to create and delete batch accounts, read and regenerate batch account keys, and manage batch account storage.
+
+```bash
+pip install azure-mgmt-batch
+```
+
+### Example
+Create an Azure Batch account and configure a new application and Azure storage account for it.
+
+```python
+from azure.mgmt.batch import BatchManagementClient
+from azure.mgmt.resource import ResourceManagementClient
+from azure.mgmt.storage import StorageManagementClient
+
+LOCATION = 'eastus'
+GROUP_NAME ='batchresourcegroup'
+
+# Create Resource group
+print('Create Resource Group')
+resource_client.resource_groups.create_or_update(GROUP_NAME, {'location': LOCATION})
+
+# Create a storage account
+storage_params = azure.mgmt.storage.models.StorageAccountCreateParameters(
+    location=LOCATION,
+    account_type=azure.mgmt.storage.models.AccountType.standard_lrs
+)
+creating = storage_client.storage_accounts.create(
+    RESOURCE_GROUP,
+    'pythonstorageaccount',
+    storage_params
+)
+creating.wait()
+
+# Create a Batch Account, specifying the storage account we want to link
+storage_resource = '/subscriptions/{}/resourceGroups/{}/providers/Microsoft.Storage/storageAccounts/{}'.format(
```

Comment:
Using the previous creation, you can just do `storage_account.id`

So it depends what we want to show:
- You just created the stroage account? Use `storage_account.id`
- You already have the storage but can do a SDK call? Use `storage_client.accounts.get("pythonstorageacount").id`
- You already have the storage but can't do a SDK call? Build yourself the id using this template: .......